vars/centos-8.yml

```diff
@@ -14,5 +14,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+_rpm_repository:
+  - name: "https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm"
```

Comment:
Use this URL: https://github.com/alvistack/ansible-role-bootstrap/blob/master/vars/centos-8.yml#L18